src/components/Assets.tsx

```diff
@@ -211,12 +218,12 @@ export const Assets = ({
 
   // Withdraw asset
   useEffect(() => {
-    if (
-      sep24WithdrawAsset.status === ActionStatus.SUCCESS &&
-      sep24WithdrawAsset.data.currentStatus === "completed"
-    ) {
+    if (sep24WithdrawAsset.status === ActionStatus.SUCCESS) {
       dispatch(resetSep24WithdrawAssetAction());
-      handleRefreshAccount();
+
+      if (sep24WithdrawAsset.data.currentStatus === "completed") {
```

Comment:
`completed` should probably be a const

Reply:
Created [an issue](https://github.com/stellar/nebula/issues/134) to do it for all transaction statuses.